<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QDialog" name="SetupWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emu64 Einstellungen</string>
  </property>
  <widget class="QTabWidget" name="SetupTab">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="EmualtionTab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Emulation</string>
    </attribute>
    <widget class="QGroupBox" name="FullscreenGroup_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>SID</string>
     </property>
     <widget class="QComboBox" name="FirstSidTyp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="SecondSidTyp">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="SecondSidAddress">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sid1_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1. SID</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SecondSidEnable">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2. SID</string>
      </property>
     </widget>
     <widget class="QLabel" name="sid_io_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IO Bereich</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SidCycleExactEnable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Zyklengenaue Emulation</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SidFilterEnable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter Emulation</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Sid6ChannelMode">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>6 Kanal Modus</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>381</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>ROM Set's</string>
     </property>
     <widget class="QComboBox" name="SelectRomSet">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="NewRomSet">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Neu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteRomSet">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>LÃ¶schen</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>VIC</string>
     </property>
     <widget class="QCheckBox" name="VicSpriteViewEnable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Sprites anzeigen</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="VicBorderViewEnable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Rahmen anzeigen</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="VicSprSprCollisionEnable">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Sprite - Sprite Kollision</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="VicSprBgrCollisionEnable">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Sprite - Hintergrund Kollision</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="VicGreyDotsEnable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Grey Dots anzeigen</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="SoundTab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Sound</string>
    </attribute>
    <widget class="QDial" name="SIDVolume">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchTarget">
      <double>3.600000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SID Volume</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SIDVolumeOut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="GrafikTab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Grafik</string>
    </attribute>
    <widget class="QLabel" name="label_Farbmodus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>C64 Farbmodus:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="C64Farbmodus">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Setzt das C64 Farbschema wenn kein PAL ausgewÃ¤hlt wurde.</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="WindowGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Fenstermodus</string>
     </property>
     <widget class="QCheckBox" name="WPal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>177</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Schaltet die Video CRT Emulation der Grafikausgabe ein. (Sollte vorrangig benutzt werden)</string>
      </property>
      <property name="text">
       <string>Video CRT Ausgabe</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="WDouble">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>177</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Verdoppelt die Textur in der Breite und in der HÃ¶he. (SchÃ¤rferes Bild beim Skalieren)</string>
      </property>
      <property name="text">
       <string>TexturgrÃ¶Ãe verdoppeln</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="WAspectRatio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>177</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Erzwingt ein Bildschirm SeitenverhÃ¤ltnis von 4:3</string>
      </property>
      <property name="text">
       <string>4:3 SeitenverhÃ¤ltnis</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="AutoMouseHide">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Blendet den Mauszeiger nach einer einstellbaren Zeit automatisch aus.</string>
      </property>
      <property name="text">
       <string>Mauszeiger ausblenden</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="AutoMouseHideTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Zeit in Sekunden.</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sek.</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="FullscreenGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>381</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Vollbildmodus</string>
     </property>
     <widget class="QCheckBox" name="FAspectRatio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>177</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Erzwingt ein Bildschirm SeitenverhÃ¤ltnis von 4:3</string>
      </property>
      <property name="text">
       <string>4:3 SeitenverhÃ¤ltnis</string>
      </property>
     </widget>
     <widget class="QComboBox" name="VideoModes">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Mode:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="DisplayList">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="screenshot_group">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Bildschirmfotos vom C64</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Format</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ScreenshotFormat">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Hier kann man das Screenshot Format einstellen.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ResetSShotCounter">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Setzt den ZÃ¤hler fÃ¼r die angehÃ¤ngte Zahl an der Screenshot Datei zurÃ¼ck.</string>
      </property>
      <property name="text">
       <string>Counter ZurÃ¼cksetzen</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="WFilter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Schaltet einen Bilineare Filterung mit zu. (Das Bild wird etwas verwaschener)</string>
     </property>
     <property name="text">
      <string>Filter aktivieren</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Vsync">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertikal Sync</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SteuerungTab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>Steuerung</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Virtuelle - Joystick's</string>
     </property>
     <widget class="QTableWidget" name="VJoySlots">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>377</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Maus 1351</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QComboBox" name="MausPort">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Hier kann man das Screenshot Format einstellen.</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Aktivieren der Maus im C64 Bildschirm mit Strg+Li. Maustaste</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="TestTab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Testbereich</string>
    </attribute>
    <attribute name="toolTip">
     <string>Hier kÃ¶nnen neue Funktionen getestet werden.</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Emu64 Timing</string>
     </property>
     <widget class="QSpinBox" name="cycles_per_second">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="default_50hz">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>50 FPS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="default_pal_c64">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>50,125 FPS</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
